pkgs/by-name/db/dbdev/package.nix

```
@@ -0,0 +1,44 @@
+{
+  lib,
+  SystemConfiguration,
+  fetchFromGitHub,
+  gitUpdater,
+  openssl,
+  pkg-config,
+  rustPlatform,
+  stdenv,
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "dbdev";
+  version = "0.1.6";
+  sourceRoot = "${src.name}/cli";
+
+  src = fetchFromGitHub {
+    owner = "supabase";
+    repo = "dbdev";
+    rev = "v${version}";
+    hash = "sha256-G9U+dhSJI03n0lbBEVOrzslj8TlmiRwrg8DQ7hGsqIo=";
+  };
+
+  cargoHash = "sha256-v7QHUCKJjfTiG8ZylBmJ0rWRaeU9imGJBQhcESzZyUM=";
+
+  nativeBuildInputs = [ pkg-config ];
+
+  # Needed to get openssl-sys to use pkg-config.
+  OPENSSL_NO_VENDOR = 1;
+
+  buildInputs = [ openssl ] ++ lib.optionals stdenv.isDarwin [ SystemConfiguration ];
```

```suggestion
  buildInputs = [ openssl ]
    ++ lib.optionals stdenv.isDarwin [
      darwin.apple_sdk.frameworks.SystemConfiguration
    ];
```

You can pass `darwin` as an input and the add `SystemConfiguration` like this. See other packages in `pkgs/by-name`.

(I think `nixfmt` would make it like this)